core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -1837,57 +1837,6 @@ public void testCancelOneQueryOutOfMultipleRunningOnes() throws Exception {
         });
     }
 
-    @Test
-    public void testCancelQueryThatReusesCircuitBreakerFromPreviousConnection() throws Exception {
```

Comment:
Why the test is deleted?

Reply:
Can we have a test that tests that the Circuit Breaker is not used from the previous connection instead?

Reply:
Re-added it and fixed stability issue . 

I removed it earlier because : 
- circuit breakers belong to pg context now and aren't shared via registry 
- same backend pid doesn't reliably mean the same circuit breaker (it is the case only if previous connection was cached in factory)